languages/go/tests/enforcement_test.go

```diff
@@ -220,7 +220,7 @@ func (a TestAdapter) BuildQuery(filter *types.Filter) (interface{}, error) {
 	return nil, nil
 }
 
-func (a TestAdapter) ExecuteQuery(query interface{}) (interface{}, error) {
+func (a TestAdapter) ExecuteQuery(query interface{}) ([]interface{}, error) {
```

Comment:
I think we want this

Reply:
yeah I agree even if its a little weird to create, otherwise can't do the relation on bound thing reliably